src/js/game/LevelMVC/LevelModel.js

```diff
@@ -49,6 +49,9 @@ module.exports = class LevelModel {
       if (this.actionPlane.getBlockAt(position).blockType.substring(0,12) === "redstoneWire") {
         this.actionPlane.determineRedstoneSprite(position);
       }
+      if (this.actionPlane.getBlockAt(position).blockType.startsWith("rails") && this.actionPlane.getBlockAt(position).blockType !== "railsRedstoneTorch") {
```

Comment:
similarly here, let's just use `.isRail`. Keep all the knowledge about what the blockType strings mean inside of LevelBlock